paddle/go/cgo_test.cc

```diff
@@ -0,0 +1,5 @@
+#include <iostream>
+#include "gtest/gtest.h"
+#include "libadder.h"
+
+TEST(Cgo, Invoke) { EXPECT_EQ(GoAdder(30, 12), 42); }
```

Comment:
`set (CMAKE_EXE_LINKER_FLAGS "-pthread")` need to be added for the linking of this file to pass.